cli/cmd/complition.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"os"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(completionCmd)
+}
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script for your shell",
+	Long: `To load completions:
+Bash:
+  $ source <(pscale completion bash)
+  # To load completions for each session, execute once:
+  # Linux:
+  $ pscale completion bash > /etc/bash_completion.d/pscale
+  # macOS:
+  $ pscale completion bash > /usr/local/etc/bash_completion.d/pscale
+Zsh:
+  # If shell completion is not already enabled in your environment,
+  # you will need to enable it.  You can execute the following once:
+  $ echo "autoload -U compinit; compinit" >> ~/.zshrc
+  # To load completions for each session, execute once:
+  $ pscale completion zsh > "${fpath[1]}/_yourprogram"
+  # You will need to start a new shell for this setup to take effect.
+fish:
+  $ pscale completion fish | source
+  # To load completions for each session, execute once:
+  $ pscale completion fish > ~/.config/fish/completions/pscale.fish
+PowerShell:
+  PS> pscale completion powershell | Out-String | Invoke-Expression
+  # To load completions for every new session, run:
+  PS> pscale completion powershell > pscale.ps1
+  # and source this file from your PowerShell profile.
+`,
```

Comment:
Ohh got it, Its the default only will change the references (pscale) even if we default we have to add the (rill) ref, But yeah will add this link comment so that handy for dev to look for further addition